src/ol_concourse/lib/resources.py

```diff
@@ -183,6 +183,29 @@ def pulumi_provisioner(
     )
 
 
+def pypi(
+    name: Identifier,
+    package_name: str,
+    username: str,
+    password: str,
```

Comment:
If you want it should be safe to add default values of the var that we are populating below.
```suggestion
    username: str = "((pypi_creds.username))",
    password: str = "((pypi_creds.password))",
```